packages/mgt-components/src/components/mgt-people/mgt-people.ts

```diff
@@ -407,8 +428,31 @@ export class MgtPeople extends MgtTemplatedComponent {
           this.people = await findGroupMembers(graph, null, this.groupId, this.showMax, PersonType.person);
         } else if (this.userIds) {
           this.people = await getUsersForUserIds(graph, this.userIds);
+          if (this._fallbackDetails) {
+            // replace null people with fallback details
+            this.people = this.people.map((p, i) => {
+              if (p) {
+                return p;
+              } else if (i < this._fallbackDetails.length) {
+                return this._fallbackDetails[i];
+              }
+              return null;
+            });
+          }
         } else if (this.peopleQueries) {
           this.people = await getUsersForPeopleQueries(graph, this.peopleQueries);
+
+          if (this._fallbackDetails) {
```

Comment:
Try to avoid duplicating this code and refactor the code here. Ex:

```ts
else if (this.userIds || this.peopleQueries) {
	this.people = await (this.userIds ? getUsersForUserIds(graph, this.userIds) : getUsersForPeopleQueries(graph, this.peopleQueries))

	if (this._fallbackDetails) {
		...
	}
} ...